_oasis

```diff
@@ -1,45 +0,0 @@
-OASISFormat:  0.4
-Name:         gperftools
-Version:      0.3
-License:      LGPL-2.1 with OCaml linking exception
-Authors:      ygrek
-Copyrights:   (C) 2012 ygrek
-Homepage:     http://ygrek.org.ua/p/ocaml-gperftools/
-BuildTools:   ocamlbuild
-Plugins:      DevFiles (0.4), META (0.4)
-
-Synopsis: Bindings to gperftools
-Description:
-  gperftools <https://code.google.com/p/gperftools/> library
-  provides fast, multi-threaded malloc() and some nifty performance
-  analysis tools.
-  These bindings are useful to monitor memory usage and control
-  behaviour of malloc() when the program is linked against tcmalloc.
-
-Flag strict
-  Description: Strict compile-time checks
-  Default: true
-
-Library gperftools
-  Path: src
-  Modules: Gperftools
-  if flag(strict) && ccomp_type(cc)
-    CCOpt: -std=c89 -pedantic -Wno-long-long -Wextra
```

Comment:
it is sad to see these options gone

Reply:
I can make add them permanently to `c_flags` if you want. I'm not sure there's a way to make them optional with dune.